deployer/src/handlers/mod.rs

```diff
@@ -56,15 +54,16 @@ pub struct PaginationDetails {
 #[derive(Clone)]
 pub struct RouterBuilder {
     router: Router,
-    _project_name: ProjectName,
+    // might be used for tracing instruments?
+    _project_name: String,
```

Comment:
No idea. Should be better to delete

Reply:
Don't want to risk breaking anything with this PR. No risk in keeping it.

Reply:
Fair enough